code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -787,7 +787,7 @@ obj/item/asteroid/basilisk_hide/New()
 		for(var/i = 0 to rand(24,46))
 			new possessed_ore(src.loc)
 
-	new /obj/item/weapon/vinyl/rock(src.loc) //It is a rock monster after all
```

Comment:
👍

Reply:
I like the pun but every juke already has rock and to make this vinyl something of value you'd need to remove rock from the jukeboxes which is dumb.
Curse whoever did this originally.

Reply:
I was the one who did this originally. I was mistaken and thought rock was only on the DJ satellite.